src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -0,0 +1,208 @@
+import _has from 'lodash/has';
+import _isNil from 'lodash/isNil';
+import EventBus from '../lib/EventBus';
+import NavigationLibrary from '../navigationLibrary/NavigationLibrary';
+import RadarTargetCollection from './RadarTargetCollection';
+import { EVENT } from '../constants/eventNames';
+import { DECIMAL_RADIX } from '../constants/globalConstants';
+import { THEME } from '../constants/themes';
+
+export default class ScopeModel {
+    constructor() {
+        this._eventBus = EventBus;
+        this._navigationLibrary = NavigationLibrary;
+        // TODO: Use this!
+        this._sectorCollection = [];
```

Comment:
doc block and default to `null` if this becomes an instance of a class

Reply:
done